src/delta5node/delta5node.ino

```diff
@@ -428,7 +435,7 @@ byte i2cHandleRx(byte command) { // The first byte sent by the I2C master is the
 		case WRITE_CALIBRATION_MODE:
 			if (readAndValidateIoBuffer(WRITE_CALIBRATION_MODE, 1)) {
 				state.calibrationMode = ioBufferRead8();
-				state.rssiTrigger = state.rssi - settings.calibrationOffset;
+				state.rssiTrigger = state.nodeRssiPeak - settings.calibrationOffset;
```

Comment:
This looks like it assumes that you always want to set the trigger value based on peak RSSI, not the peak of the current quad in the heat.